galley/pkg/config/analysis/diag/level.go

```diff
@@ -14,17 +14,21 @@
 
 package diag
 
+import (
+	"strings"
+)
+
 // Level is the severity level of a message.
 type Level struct {
 	sortOrder int
 	name      string
 }
 
-func (l Level) String() string {
+func (l *Level) String() string {
 	return l.name
 }
 
-func (l Level) IsWorseThanOrEqualTo(target Level) bool {
+func (l *Level) IsWorseThanOrEqualTo(target Level) bool {
```

Comment:
We can go back to value receivers here, and get rid of the downstream pointer changes we don't need anymore.